### PR TITLE
fix: Add Acr And Storage Ingredient

### PR DESCRIPTION
**Pull request by csperbeck**
Azure Container Registry and Storage basic ingredients without value reflection.

### CODE REVIEW

File: ingredient/ingredient-container-reg/src/functions.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import {BaseUtility, IngredientManager} from '@azbake/core'
+
+export class ContainerRegUtils extends BaseUtility {
+
+    public get_profile(): string {
```
Comment (whilke) [inline ingredient/ingredient-container-reg/src/functions.ts]: Rename to create_resource_name(), which I believe is the standard pattern for this.

File: ingredient/ingredient-container-reg/src/functions.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+        let util = IngredientManager.getIngredientFunction("coreutils", this.context)
+        const acr_profile = util.create_resource_name("acr", null, false);
+        return acr_profile;
+    }
+    
+    public get_region(): string {
```
Comment (whilke) [inline ingredient/ingredient-container-reg/src/functions.ts]: Remove this, don't need to create a method that is a duplicate off coreutil

File: ingredient/ingredient-container-reg/src/functions.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        let util = IngredientManager.getIngredientFunction("coreutils", this.context)
+        const acr_region = util.current_region()
+        return acr_region
+    }
+
+    public get_primary_region(): string {
```
Comment (whilke) [inline ingredient/ingredient-container-reg/src/functions.ts]: Remove this, don't need to create a method that is a duplicate off coreutil

File: ingredient/ingredient-container-reg/src/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (whilke) [inline ingredient/ingredient-container-reg/src/tsconfig.json]: Can you move this up to the root folder, and update src to point to src folder. We refactored all the projects for tsconfig to live at project root. You can see that in any ingredient under latest master.

File: ingredient/ingredient-storage/src/functions.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+export class StorageUtils extends BaseUtility {
+
+    public get_profile(): string {
+        let util = IngredientManager.getIngredientFunction("coreutils", this.context)
+        const st_profile = util.create_resource_name("st", null, false);
+        return st_profile;
+    }
+    
+    public get_region(): string {
```
Comment (whilke) [inline ingredient/ingredient-storage/src/functions.ts]: Remove this, don't need to create a method that is a duplicate off coreutil

File: ingredient/ingredient-storage/src/functions.ts [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        let util = IngredientManager.getIngredientFunction("coreutils", this.context)
+        const st_region = util.current_region()
+        return st_region
+    }
+
+    public get_primary_region(): string {
```
Comment (whilke) [inline ingredient/ingredient-storage/src/functions.ts]: Remove this, don't need to create a method that is a duplicate off coreutil

File: ingredient/ingredient-storage/src/functions.ts [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import {BaseUtility, IngredientManager} from '@azbake/core'
+
+export class StorageUtils extends BaseUtility {
+
+    public get_profile(): string {
```
Comment (whilke) [inline ingredient/ingredient-storage/src/functions.ts]: Rename to create_resource_name()

File: ingredient/ingredient-storage/src/tsconfig.json [hunk 8]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (whilke) [inline ingredient/ingredient-storage/src/tsconfig.json]: move this file up to project root, and fix include path to point to src.

Comment (csperbeck) [reply ingredient/ingredient-storage/src/tsconfig.json]: Moved tsconfig and fixed references.

File: ingredient/ingredient-storage/src/functions.ts (hunk 7 above)
Comment (csperbeck) [reply ingredient/ingredient-storage/src/functions.ts]: Renamed to create_resource_name()

File: ingredient/ingredient-storage/src/functions.ts (hunk 6 above)
Comment (csperbeck) [reply ingredient/ingredient-storage/src/functions.ts]: Removed function.

File: ingredient/ingredient-storage/src/functions.ts (hunk 5 above)
Comment (csperbeck) [reply ingredient/ingredient-storage/src/functions.ts]: Removed function.

File: ingredient/ingredient-container-reg/src/tsconfig.json (hunk 4 above)
Comment (csperbeck) [reply ingredient/ingredient-container-reg/src/tsconfig.json]: Moved tsconfig and fixed references.

File: ingredient/ingredient-container-reg/src/functions.ts (hunk 3 above)
Comment (csperbeck) [reply ingredient/ingredient-container-reg/src/functions.ts]: removed function

File: ingredient/ingredient-container-reg/src/functions.ts (hunk 1 above)
Comment (csperbeck) [reply ingredient/ingredient-container-reg/src/functions.ts]: Renamed to `create_resource_name()`

File: ingredient/ingredient-container-reg/src/functions.ts (hunk 2 above)
Comment (csperbeck) [reply ingredient/ingredient-container-reg/src/functions.ts]: Removed function.

**Pull request merged**